lib/client.js

```diff
@@ -43,9 +43,9 @@ $(document).ready(function () {
     var msg = JSON.parse(message);
     var html = '<li class=\'row\'>';
 
+    html += '<span class=\'name\'>' + msg.n + '</span>';
     html += '<small class=\'time\'>' + getTime(msg.t) + ' </small>';
-    html += '<span class=\'name\'>' + msg.n + ': </span>';
-    html += '<span class=\'msg\'>' + msg.m + '</span>';
+    html += '<p class=\'msg\'>' + msg.m + '</p>';
```

Comment:
@nelsonic swapping the order of the time and name was mostly to mirror other chat apps i've used (Slack, HipChat).  I was also toying with the idea of right aligning the timestamp on tablet and desktop (would have required a `float` if I left the name and timestamp as they were), but decided against it until I got some feedback 😅 

I can change the order back before the PR gets merged if that's the preferred layout.